csharp/src/Ice/Connection.cs

```diff
@@ -379,10 +379,9 @@ public void ThrowException()
         /// <returns>The description of the connection as human readable text.</returns>
         public override string ToString() => _transceiver.ToString()!;
 
-        /// <summary>Returns the connection type. This corresponds to the endpoint type, i.e., "tcp", "udp", etc.
-        /// </summary>
-        /// <returns>The type of the connection.</returns>
-        public string Type() => _transceiver.Transport; // No mutex lock, _type is immutable.
+        /// <summary>Returns the transport's name. This corresponds to "tcp", "udp", etc.</summary>
+        /// <returns>The transport name.</returns>
+        public string TransportName => _transceiver.TransportName;
```

Comment:
Do we need this, or can we just use `Endpoint.TransportName`?

Reply:
Removed.